WebContent/view/Home.jsp

```diff
@@ -0,0 +1,73 @@
+<%@ page import="java.util.List"%>
+<%@ page import="java.util.ArrayList"%>
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+<%@ page import="com.codeenginestudio.bookManagement.models.Book"%>
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<link rel="stylesheet"
+	href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
```

Comment:
@HuongNguyenThiThu @thuycao625 Should not use the direct link to library. We should download it and put into our folder.